docs/foundation/logos.mdx

```diff
@@ -2,53 +2,102 @@
 name: Logos
 menu: Foundation
 ---
-import LogoBox from './js/logobox'
 
-import MayaDataLight from '../../resources/logos/mayadata-light.svg'
-import MayaDataDark from '../../resources/logos/mayadata-dark.svg'
+import LogoBox from "./js/logobox";
 
-import MDAPLight from '../../resources/logos/mdap-light.svg'
-import MDAPDark from '../../resources/logos/mdap-dark.svg'
+import MayaDataLight from "../../resources/logos/mayadata/mayadata-white.svg";
+import MayaDataDark from "../../resources/logos/mayadata/mayadata-dark.svg";
 
-import MayaOnlineLight from '../../resources/logos/mayaonline-light.svg'
-import MayaOnlineDark from '../../resources/logos/mayaonline-dark.svg'
+import OpenEBSEnterprisePlatformLight from "../../resources/logos/openebs-enterprise-platform/openebs-enterprise-platform-white.svg";
+import OpenEBSEnterprisePlatformDark from "../../resources/logos/openebs-enterprise-platform/openebs-enterprise-platform-dark.svg";
 
-import OpenEBSLight from '../../resources/logos/openebs.svg'
-import OpenEBSDark from '../../resources/logos/openebs-dark.svg'
+import DirectorOnlineLight from "../../resources/logos/director-online/directoronline-white.svg";
+import DirectorOnlineDark from "../../resources/logos/director-online/directoronline-dark.svg";
 
-import LitumsLight from '../../resources/logos/litmus.svg'
-import LitumsDark from '../../resources/logos/litmus-dark.svg'
+import DirectorOnPremLight from "../../resources/logos/director-onprem/directoronprem-white.svg";
+import DirectorOnPremDark from "../../resources/logos/director-onprem/directoronprem-dark.svg";
 
+import OpenEBSLight from "../../resources/logos/openebs/openebs-white.svg";
+import OpenEBSDark from "../../resources/logos/openebs/openebs-dark.svg";
+
+import LitumsLight from "../../resources/logos/litmus/litmus-white.svg";
+import LitumsDark from "../../resources/logos/litmus/litmus-dark.svg";
 
 # Logos
+
+#### Notes\*
+
+> When you download any of logos, you will get PNG and SVG version of that logo in colored, white and (grey/black) formats.
```

Comment:
Each download includes all approved formats and color options.

Reply:
or
On downloading any of the logos, you will get PNG and SVG version of that logo in (colored, white and grey/black) formats.

Reply:
@IsAmrish Each download includes all approved formats (PNG,SVG) and color options (color,white, grey,black)